src/site.webmanifest

```diff
@@ -0,0 +1,8 @@
+{
+    "icons": [{
+        "src": "icon.png",
+        "sizes": "192x192",
+        "type": "image/png"
+    }],
+    "start_url": "/"
```

Comment:
> I think it's a bit risky to default to "/"

@marcoscaceres Good point.

If `start_url` is not provided (if I understand correctly), it will default to the _"document URL"_? So, basically where the file that includes the manifest will be placed?